fetch.bs

```diff
@@ -564,6 +558,9 @@ consistency.
 
 <h4 id=terminology-headers>Headers</h4>
 
+<p class=note>HTTP generally refers to headers as ‘fields’ or ‘header fields’. The web platform uses
+the more colloquial term. [[HTTP]]
```

Comment:
nit: I recognize that this is in the "Headers" section, but could we specify what the colloquial term is?

```suggestion
the more colloquial term  ‘header’. [[HTTP]]
```